spec/models/job_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'spec_helper'
+
+describe Job do
+  it_behaves_like 'an active resource model'
+
+  it { should respond_to :template_id }
+
+  describe '.new_from_template' do
+    let(:fake_template) { double(:fake_template, environment: 'some environment stuff', id: 9) }
+
+    it 'instantiates itself with attributes from the template' do
+      instance = described_class.new_from_template(fake_template)
+      expect(instance.override.environment).to eq 'some environment stuff'
+      expect(instance.template_id).to eq 9
+    end
+  end
+
+  describe '.nested_create' do
+    let(:attrs) do
+      HashWithIndifferentAccess.new({
+        'template_id' => '9',
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.